public/data/TusharAgarwal18.json

```
@@ -0,0 +1,13 @@
+{
+  "name": "Tushar Agarwal",
+  "type": "Personal",
+  "bio": "Programmer",
+  "avatar": "https://github.com/TusharAgarwal18.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/TusharAgarwal18",
+      "icon": "github"
+    },
```

Please remove this trailing comma from line no. `11`.
```suggestion
    }
```